applications/alphaday.md

```diff
@@ -0,0 +1,195 @@
+
+Zehra Naz Hacisuleyman, Marketing
+
+Selim Cetiner, Content Specialist & Marketing Operations
+
+Michael Hagopian, QA & Partnerships
+
+### Contact
+
+- **Contact Name:** Michael Hagopian
+- **Contact Email:** mike@alphaday.com
+- **Website:** alphaday.com, app.alphaday.com
+
+### Legal Structure
+
+- **Registered Address:** Alphabox Solutions Pte. Ltd.
+- **Registered Legal Entity:** 45 North Canal Road, #01-01 Lew Building, 059301 Singapore
+
+### Team's experience
+
+Deniz has been involved in the Ethereum space since early 2016 and joined Kyber Network in 2017 as Head of Ecosystem Growth before quitting to start Alphaday full-time in 2021. Previously he worked at Thomson Reuters for a decade building financial products that rivaled the Bloomberg Terminal. He is currently also a Venture Partner at IOSG.VC.
+https://twitter.com/DenizOmer, https://www.linkedin.com/in/denizomer/ 
+
+Felipe has also been involved in Ethereum since 2016 and was a Developer Advocate at Pegasys and Consensys before quitting to work full time on Alphaday.
+https://github.com/faraggi, https://twitter.com/felipefaraggi, https://www.linkedin.com/in/faraggi/ 
+
+### Team Code Repos
+
+- Felipe - https://github.com/faraggi
+- Pablo - https://github.com/ppalomo
+- Vicente - https://github.com/v-almonacid
+- Charles - https://github.com/Xavier-Charles
+- Jonathan - https://github.com/elcharitas
+- Gideon - https://github.com/getgiddy
+
+### Team LinkedIn Profiles (if available)
+
+- Deniz - https://www.linkedin.com/in/denizomer/ 
+- Felipe - https://www.linkedin.com/in/faraggi/ 
+- Pablo - https://www.linkedin.com/in/pablo-palomo-07127711/ 
+- Vicente - https://www.linkedin.com/in/vicente-almonacid/ 
+- Charles - https://www.linkedin.com/in/charles-nwankwo-01/ 
+- Jonathan - https://linkedin.com/in/elcharitas 
+- Gideon - https://www.linkedin.com/in/getgiddy/ 
+- Zehra - https://www.linkedin.com/in/zehranazh/
+- Selim - https://www.linkedin.com/in/selim-cetiner-167364234/
+- Michael - https://www.linkedin.com/in/mikael-h-87bb4ba4/
+
+## Development Status :open_book:
+
+Product is live at: app.alphaday.com
+
+Landing Page: https://alphaday.com/ 
+Twitter: https://twitter.com/AlphadayHQ 
+Discord: https://discord.com/invite/8KSmPyT5k8 
+Github: https://github.com/AlphadayHQ 
+Linkedin: https://www.linkedin.com/company/alphaday
+Pitch Deck: https://alphaday.com/deck 
+Demo: https://www.youtube.com/watch?v=ThCd_W3rK_8 
+
+## Development Roadmap :nut_and_bolt:
+
+### Overview
+
+- **Total Estimated Duration:** 2 months
+- **Full-Time Equivalent (FTE):**  2 FTE
+- **Total Costs:** 4,000 USD
+
+### Milestone 1 — Development & Deployment of the Dashboard
+
+- **Estimated duration:** 2 months
+- **FTE:**  2
+- **Costs:** 3,000 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| 0a. | License | MIT |
+| 0b. | Documentation | Not applicable to Alphaday, as we have no Substrates nodes or sending functionality on the app |
```

Comment:
Documentation is always required for development milestones, regardless of the tech stack. At the very least a GH readme, but it's nice to see user-facing documentation as well, for front-end deliverables.